mitmproxy/web/app.py

```diff
@@ -298,6 +292,22 @@ def post(self, flow_id):
 
 class FlowContent(RequestHandler):
 
+    def post (self, flow_id, message):
+        # handle request later now just change response content
+
+        flow = self.flow
+        flow.backup()
+        content = ''
+        if (len(self.request.files.values()) > 0):
+            content = self.request.files.values()[0][0]["body"]
+        elif (len(self.request.arguments) > 0):
+            content = self.request.arguments['file']
+
```

Comment:
is not working as i want, do you have any hints how to set the content, I have problems with gzip, when i upload something i need to disable gzip to view the content